### ISSUE TITLE
`rustc-perf` fails to build after #107956 

### ISSUE THREAD

**Issue by KittyBorgX** [rust-lang/rust#108258]
`rustc-perf` fails to build after #107956 

It appears broken in https://perf.rust-lang.org/status.html 

Error: 
```
thread 'main' panicked at 'self.symlink_file(&rustfmt_path, &legacy_rustfmt) failed with File exists (os error 17)', download.rs:346:17
       note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

Possible Workaround - Comment out 
```rs
       #[cfg(not(windows))]
        {
            let legacy_rustfmt = self.initial_rustc.with_file_name(exe("rustfmt", host));
            if !legacy_rustfmt.exists() {
                t!(self.symlink_file(&rustfmt_path, &legacy_rustfmt));
            }
        }
```
in `src/bootstrap/download.rs:342` to avoid symlinking `rustfmt` since are explicitly copying the `rustfmt` binary in #107956 

zulip thread : https://rust-lang.zulipchat.com/#narrow/stream/247081-t compiler.2Fperformance/topic/Bootstrap.20benchmarking.20broken

cc : @jyn514 @Kobzol

**Comment by KittyBorgX** [comment]
As per zulip discussions, the pr will be reverted (for now) 
I will release my assignment since there's nothing more that I can do.

**Comment by lqd** [comment]
This was fixed by #108302
